### PR TITLE
Rewrite On-Site welcome email

### LINKED ISSUE #16: Let's copy a signup flow as that makes devs like Ben happy

### PR DESCRIPTION
**Pull request by nexdrew**
Fixes #16, with a focus on "first 10 minute UX".

Content change only. Still only using `text/plain` email, and the data/context fields expected by the template are unchanged, namely:

- `name`: customer name
- `email`: customer's billing email
- `license_key`: customer's generated license key
- `support_email`: support@npmjs.com

Email generation tested locally using SES.

cc @bcoe

### CODE REVIEW

**Comment by nexdrew** [comment]
### Sample email in OS X Mail app:
screen shot 2015-12-30 at 9 34 11 am

### Sample email in Gmail web UI:
screen shot 2015-12-30 at 9 38 35 am

File: enterprise-verification.text.hbs [hunk 1]
```diff
@@ -1,28 +1,71 @@
 Hi {{name}},
 
-Thanks for trying out npm On-Site!
+Thanks for trying out npm On-Site - your own private npm behind the firewall!
 
-To get started, make sure you have a machine that meets the installation requirements:
+For your record, here is your license information:
```
Comment (bcoe) [inline enterprise-verification.text.hbs]: for your records?

**Comment by bcoe** [comment]
@nexdrew I appreciate how to the point the email now is, and that it links out to well-edited tutorials for each step (rather than overwhelming someone with a wall of words).

I say :shipit: once @ashleygwilliams signs off.

File: enterprise-verification.text.hbs (hunk 1 above)
Comment (nexdrew) [inline enterprise-verification.text.hbs]: Or maybe just leave that part out altogether? i.e. `Here is your license information:`

**Comment by nexdrew** [comment]
@ashleygwilliams If you get a chance, would you mind giving this email copy change a quick once over? Just want to make sure nothing jumps out at you. Not super urgent.

**Comment by rod11** [comment]
@nexdrew This email is fantastic!!!   Really like how clear it is and how cleanly it lays out the steps someone needs to go through.   :+1: :+1:

**Comment by ashleygwilliams** [comment]
was this reviewed by @kaseybyrne or @jec? 

i think it is not acceptable to merge it without their feedback.

**Comment by nexdrew** [comment]
@ashleygwilliams Sorry, I assumed Rod's feedback qualified it as acceptable. Is there a documented protocol for these kind of changes? I would gladly follow it if there is. Otherwise, just trying to get things done. I can always revert. It has not been released yet.

**Comment by cowperthwait** [comment]
I’d like to know how this integrates with [generate HTML transactional emails using Nick’s templates #284](https://github.com/npm/issue-tracker/issues/284 "generate HTML transactional emails using Nick’s templates #284"); our momentum has been towards a single unified communication system.

**Comment by ashleygwilliams** [comment]
marketing gets reviewed by marketing. this is very clearly marketing.

**Comment by kaseybyrne** [comment]
Please revert it. Jon & I both have input. thanks.

**Comment by nexdrew** [comment]
@kaseybyrne I will revert.

@cowperthwait It doesn't change anything that is currently set up for our email delivery system in `newww` - doesn't help and doesn't hurt. We currently don't use any HTML emails defined in this repo, but we probably should start doing so to track things like link clicks. If something else needs to change, I will gladly look into it.

**Comment by nexdrew** [comment]
The changes from this PR have been undone. I apologize for the mess.

**Comment by nexdrew** [comment]
Replaced by #21.

**Pull request merged**